requirements.txt

```diff
@@ -1,3 +1,4 @@
+flake8==7.0.0
```

Comment:
flake8 requirement is in `requirements-test.txt`

Reply:
Ah - I see. I hadn't noticed that one.